README.md

```diff
@@ -18,9 +18,72 @@ Or install it yourself as:
 
     $ gem install grpc-kit
 
-## Usage
+## Documentation
 
-TODO: Write usage instructions here
+- [`GRPC::Kit::Communication::Resilient`](#grpckitcommunicationresilient)
+- [`GRPC::Kit::Logger`](#grpckitlogger)
+- [`GRPC::Kit::Queue::Publisher`](#grpckitqueuepublisher)
+- [`GRPC::Kit::Queue::Worker`](#grpckitqueueworker)
+
+### GRPC::Kit::Communication::Resilient
+
+Sometimes, GRPC and Google::Cloud raises _unavailable error_, but you can retry and all works fine. Based on [Datastore documentation](https://cloud.google.com/appengine/articles/handling_datastore_errors#timeouts-due-to-datastore-issues) we implemented a _helper_ to _resilient communication_.
+
+Example:
+
+```ruby
+pubsub = Google::Cloud::Pubsub.new
+topic  = pubsub.create_topic('topic_name')
+topic.publish('message') # receives Google::Cloud::UnavailableError
+
+# But you can uses resilient communication
+include GRPC::Kit::Communication::Resilient
+topic = resilient { topic.publish('message') }
+
+# And you can sets the limit of repetition...
+topic = resilient(limit: 5) { topic.publish('message') }
+```
+
+### GRPC::Kit::Logger
+
+By default all logs sent to `GRPC.logger` are ignored. But `GRPC::Kit::Logger` configure automatically logs to use _STDOUT_.
```

Comment:
Acho que a ordem das palavras é um pouco diferente em inglês [1]:
But `GRPC::Kit::Logger` **automatically** configure**s**  logs to use _STDOUT_.

[1] https://translate.google.com/#auto/en/ele%20configura%20automaticamente%20logs%20para%20usar